frontend/package.json

```diff
@@ -23,20 +23,27 @@
     "typescript": "^3.8.2"
   },
   "scripts": {
-    "start": "parcel serve index.html --port 2345 --hmr-port 2346 --no-source-maps",
-    "build": "rm dist/* -rf; parcel build index.html --no-source-maps",
+    "start": "parcel serve index.html --port 2345 --hmr-port 2346",
+    "build": "rm dist/*; parcel build index.html",
     "test": "jest"
   },
   "dependencies": {
+    "@babel/plugin-transform-flow-strip-types": "^7.10.4",
```

Comment:
Why is this added? There are no flow types are there?

Reply:
probably can be removed. I was trying to solve my compilation issues at first installing what it seemed missing libraries, according to the error. I will check it.

Reply:
solved